packages/default-theme/components/SwTopNavigation.vue

```diff
@@ -13,14 +13,27 @@
       <nuxt-link
         class="sf-header__link"
         :to="$i18n.path(getCategoryUrl(category))"
+        >{{ category.name }}</nuxt-link
       >
-        {{ category.name }}
-      </nuxt-link>
       <SwMegaMenu
         :category="category"
         :visible="currentCategoryName && category.name === currentCategoryName"
       />
     </SfHeaderNavigationItem>
+
+    <SfHeaderNavigationItem
+      class="sf-header__link"
+      v-if="unvisibleCategories"
+      @mouseover="changeCurrentCategory('categories')"
+      @mouseleave="changeCurrentCategory(null)"
+      @keyup.tab="changeCurrentCategory('categories')"
+    >
+      <span class="sf-header__link">More</span>
```

Comment:
Would be great to use a new small component here like `SwTopNavigationShowMore` or something like this, then if someone wants to add some icon or change appearance, then only a small component need to be changed :)

Also `More` text should be translatable, please add English and German version

Reply:
Thanks, I moved that span to separate component and added translations in EN and DE.